ecms_base/config/install/editor.editor.basic_html.yml

```diff
@@ -5,6 +5,8 @@ dependencies:
     - filter.format.basic_html
   module:
     - ckeditor
+_core:
+  default_config_hash: AqlPmO16LvJI4D0Ih6u4GFQIzqr5OnLgAUSjcUGWk2g
```

Comment:
Did the clean-config script not remove this?

Reply:
Whoops! Didn't run it before - just did!